tests/download/test_views.py

```diff
@@ -252,7 +252,13 @@ def test_get_document_metadata_when_document_is_not_in_s3(client, store):
 
 
 class TestAuthenticateDocument:
-    def test_missing_decryption_key(self, client, store):
+    @pytest.fixture(scope="function")
+    def mock_exceeded_rate_limit(self):
```

Comment:
Do we need this fixture now? If redis is not enabled (which it now isn't for tests) then did you say the exceed_rate_limit will just return False anyway?

Reply:
Still need it for one test that checks when rate limit is exceeded - but I'll just inline it there now :)